engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkRestManager.scala

```diff
@@ -302,7 +306,7 @@ object flinkRestModel {
 
   @JsonCodec(decodeOnly = true) case class JobsResponse(jobs: List[JobOverview])
 
-  @JsonCodec(decodeOnly = true) case class JobOverview(jid: String, name: String, `last-modification`: Long, `start-time`: Long, state: JobStatus)
+  @JsonCodec(decodeOnly = true) case class JobOverview(jid: String, name: String, `last-modification`: Long, `start-time`: Long, state: String)
```

Comment:
Why not JobStatus? Now we can put there any string? :(

Reply:
I added comment